### PR TITLE
Add to introduction of UG

### PR DESCRIPTION
**Pull request by JustinPeng13**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,12 +3,22 @@ layout: page
...
+
+![App Logo](../src/main/resources/images/guest_book.png)
+
 GuestBook is a **desktop app for managing guests in a hotel,
 optimized for use via a Command Line Interface** (CLI)
 while still having the benefits of a Graphical User Interface (GUI).
 If you can type fast, GuestBook can get your
 guest management tasks done faster than traditional GUI apps.
 
+You can use GuestBook to track guest and room details such as emails, phone numbers, room numbers, dates of stay,
```
Comment (bryanhce) [inline docs/UserGuide.md]: Should this be "track guests" instead of "guest"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,12 +3,22 @@ layout: page
...
+numbers of guests, bills, requests, and whether the rooms have been cleaned. By allowing you to add, edit, delete, and
```
Comment (bryanhce) [inline docs/UserGuide.md]: Should this be 'number of guests'?

File: docs/UserGuide.md (hunk 1 above)
Comment (jehousoh) [inline docs/UserGuide.md]: Should we add name?

**Review by bryanhce — CHANGES_REQUESTED** [review]
some nits to fix!

**Review by jehousoh — CHANGES_REQUESTED** [review]
One small comment

File: docs/UserGuide.md (hunk 1 above)
Comment (JustinPeng13) [reply docs/UserGuide.md]: Modified

File: docs/UserGuide.md (hunk 2 above)
Comment (JustinPeng13) [reply docs/UserGuide.md]: Modified

File: docs/UserGuide.md (hunk 1 above)
Comment (JustinPeng13) [reply docs/UserGuide.md]: Added

**Review by jehousoh — APPROVED** [review]
lgtm!

**Pull request merged**